src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationAbstractMethodSymbol.cs

```diff
@@ -32,7 +32,7 @@ public virtual ImmutableArray<AttributeData> GetReturnTypeAttributes()
         public abstract int Arity { get; }
         public abstract bool ReturnsVoid { get; }
         public abstract bool ReturnsByRef { get; }
-        public abstract bool ReturnsByReadonlyRef { get; }
+        public abstract bool ReturnsByRefReadonly { get; }
```

Comment:
This change is ok for now

---
In reply to: [103024354](https://github.com/dotnet/roslyn/pull/17337#discussion_r103024354) [](ancestors = 103024354,102998261)